libproto/src/protos/request.proto

```diff
@@ -39,6 +39,8 @@ message Request {
         bytes transaction_proof = 21;
         // cita_getMetaData
         string meta_data = 22;
+        // eth_getBalance
+        string balance = 23;
```

Comment:
I think the variable name is weired.
The content for this variable is not `balance`.

Reply:
The name 'balance' used here is more like a 'identifier', indicting its' usage, 
and the content include all input parameters,  not the quantity of balance,
Just like code and abi above.